langchain-llamaindex-slackbot/README.md

```diff
@@ -0,0 +1,7 @@
+# Playground for docs-chat
+
+## Installation
```

Comment:
This needs some love and id add a requirements.txt to make things easier

Reply:
Added a bunch of things in there now.